data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2815,12 +2815,22 @@ spec:
                       load balancer. Deprecated: this value has been renamed to apiFloatingIP.'
                     type: string
                   machinesSubnet:
-                    description: MachinesSubnet is the UUIDv4 of an openstack subnet.
+                    description: 'MachinesSubnet is the UUIDv4 of an openstack subnet.
                       This subnet will be used by all nodes created by the installer.
                       By setting this, the installer will no longer create a network
                       and subnet. The subnet and network specified in MachinesSubnet
-                      will not be deleted or modified by the installer.
+                      will not be deleted or modified by the installer. Deprecated:
+                      Use machinesSubnets'
                     type: string
+                  machinesSubnets:
```

Comment:
See my comments here: https://github.com/openshift/enhancements/pull/1319/files#r1085865860

I think we need to discuss of the future of `machineSubnet` and eventually use FailureDomains for BYON advanced config, instead of adding `machineSubnets`. To me they both achieve BYON, so we need to converge I think.